### PR TITLE
Adding polarion IDs and test case for multi node gateway failure scenarios

### PR DESCRIPTION
**Pull request by manasagowri**
Created the following polarion test cases for tests in the given test suite.
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83591995
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83591996
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83591996

Also added test case for the polarion ID
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83589021

And updated few existing polarion IDs to the test cases defined in the given test suite.

### CODE REVIEW

**Comment by manasagowri** [comment]
@sunilkumarn417 Made changes as discussed yesterday. Please review and approve.

File: suites/reef/nvmeof/tier-2_nvmeof_4nodes_gateway_ha_tests.yaml [hunk 1]
```diff
@@ -444,13 +474,13 @@ tests:
         fault-injection-methods:                # Failure induction
           - tool: systemctl
             nodes:
-              - node6
-              - node9
-      desc: 4GW HA 2-subsystems multinode Failover and failback parallely
+              - node8
```
Comment (sunilkumarn417) [inline suites/reef/nvmeof/tier-2_nvmeof_4nodes_gateway_ha_tests.yaml:477]: add one more combination

File: suites/reef/nvmeof/tier-2_nvmeof_4nodes_gateway_ha_tests.yaml [hunk 2]
```diff
@@ -529,12 +559,13 @@ tests:
             listener_port: 4420
             node: node10
         fault-injection-methods:                # Failure induction
-          - tool: systemctl
+          - tool: daemon
             nodes:
+              - node6
```
Comment (sunilkumarn417) [inline suites/reef/nvmeof/tier-2_nvmeof_4nodes_gateway_ha_tests.yaml:564]: add one more combination

File: suites/reef/nvmeof/tier-2_nvmeof_4nodes_gateway_ha_tests.yaml [hunk 3]
```diff
@@ -371,21 +385,17 @@ tests:
...
-          - tool: systemctl
-            nodes:
-              - node7
-              - node9
           - tool: systemctl
             nodes:
               - node6
-              - node7
-      desc: 4GW HA test Single multinode sequential Failure
+              - node9
```
Comment (sunilkumarn417) [inline suites/reef/nvmeof/tier-2_nvmeof_4nodes_gateway_ha_tests.yaml:391]: add one more combination

**Comment by manasagowri** [comment]
@sunilkumarn417 Added 2 combinations for every test case. Please take a look.

**Review by sunilkumarn417 — APPROVED** [review]

**Pull request merged**